test/check-machines-snapshots

```diff
@@ -237,6 +238,11 @@ class TestMachinesSnapshots(machineslib.VirtualMachinesCase):
                     print("------ end snapshot XML -------")
                     raise
 
+            def revert(self):
+                b.click(f"#vm-subVmTest1-snapshot-{self.snap_num}-revert")
+                b.click('.pf-v5-c-modal-box__footer button:contains("Revert")')
+                b.wait_not_present('.pf-v5-c-modal-box__footer button:contains("Revert")')
```

Comment:
Hmm, this feels a little thin -- it doesn't do any checks if it actually worked (in libvirt). `testSnapshotRevert` covers this. It should also check that the snapshot goes away from the list after reverting it.

Would it be reasonable to move some of the `virsh snapshot-info` checks here, and move these out of `testSnapshotRevert`?

Reply:
> It should also check that the snapshot goes away from the list after reverting it.
No, it will stay in the list.

> Would it be reasonable to move some of the virsh snapshot-info checks here, and move these out of testSnapshotRevert?

Yes, I think so too. Also stuff from testSnapshot.

Reply:
> No, it will stay in the list.

Ah ok -- but does *anything* change in the UI? There's a "current" indicator or so? Can this be detected via the CLI somehow, or purely inside the guest VM? (which is hard to introspect)

Reply:
> Ah ok -- but does _anything_ change in the UI?

Actually, no. The snapshot was current before reverting it, and it is still current afterwards. I now think the test is fine as it is, providing a extra check that the created snapshot can actually be reverted to. testSnapshotRevert tests multiple snapshots and their relations.

Reply:
OK, let's keep that simple approach then.